AdminWebsite/AdminWebsite/Services/HearingsService.cs

```diff
@@ -415,7 +415,7 @@ public async Task<TeleConferenceDetails> GetTelephoneConferenceDetails(Guid hear
             // Map the request except the username
             var newParticipant = NewParticipantRequestMapper.MapTo(participant);
             // Judge and panel member is manually created in AD, no need to create one
-            if (participant.CaseRoleName == "Judge" || participant.CaseRoleName == "Panel Member" || participant.CaseRoleName == "Winger")
+            if (participant.CaseRoleName == "Judge" || participant.HearingRoleName == "Panel Member" || participant.HearingRoleName == "Winger")
```

Comment:
Is there case types where a PanelMember or Winger can be in different CaseRoles? This is another name for party right?

Reply:
yeah, they are another party.